publications-2015.bib

```diff
@@ -625,7 +625,16 @@ @Article{2015:holmgren.kreiss:towards
   title   = {Towards Accurate Modeling of Moving Contact Lines},
   journal = {arXiv:1510.06639},
   year    = 2015,
-  url     = {http://arxiv.org.ezproxy.library.tamu.edu/abs/1510.06639}
+  url     = {http://arxiv.org/abs/1510.06639}
+}
+
+@MastersThesis{2015:holmgren:towards,
```

Comment:
I am not sure whether a 'Licentiate thesis' in Sweden is equivalent to a PhD dissertation or Master's thesis. Do you have have an idea?

Reply:
Good enough. Not worth the time to worry about these sorts of things.

Reply:
This is correct. For completeness, a licentiate thesis is half way between M.Sc. and Ph.D. (typically obtained after 2 years of Ph.D. studies and about 2 years before the defense).